pulpcore/app/viewsets/content.py

```diff
@@ -138,3 +149,4 @@ class SigningServiceViewSet(NamedModelViewSet,
     endpoint_name = 'signing-services'
     queryset = SigningService.objects.all()
     serializer_class = SigningServiceSerializer
+    filterset_class = SigningServiceFilter
```

Comment:
How about using `filterset_fields` (https://django-filter.readthedocs.io/en/master/guide/rest_framework.html#using-the-filterset-fields-shortcut)? Then you do not need to create a separate class.
```suggestion
    filterset_fields = ['name']
```